spec/support/vcr.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.configure do |config|
+  # pass `vcr: true` to an example to run it using VCR
+  config.around(:each, vcr: true) do |example|
+    name = example.metadata[:full_description].gsub(/\W+/, "_").split("_", 2).join("/").gsub('::', '_')[0, 94]
```

Comment:
Line is too long. [110/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.